pkgs/development/libraries/agda/generic/default.nix

```diff
@@ -2,13 +2,13 @@
 
 mkDerivation rec {
   pname = "generic";
-  version = "0.1.0.1";
+  version = "0.1.0.2";
 
   src = fetchFromGitHub {
-    repo = "Generic";
     owner = "effectfully";
+    repo = "Generic";
```

Comment:
Not sure why these lines were permuted.

Reply:
(But it doesn't matter.)

Reply:
I did it on purpose, because I was of the impression that we usually list the repo owner first and the repo name second. :-)

Reply:
Yes good point. That way, it's easier to parse it into github.com/effectfully/generic.